server/sonar-server/src/main/java/org/sonar/server/platform/ws/InfoAction.java

```diff
@@ -53,11 +55,12 @@ public void define(WebService.NewController controller) {
   @Override
   public void handle(Request request, Response response) throws Exception {
     userSession.checkIsSystemAdministrator();
-    try (JsonWriter json = response.newJsonWriter()) {
-      json.beginObject();
-      systemInfoWriter.write(json);
-      json.endObject();
-    }
+    StringWriter stringWriter = new StringWriter();
+    JsonWriter json = JsonWriter.of(stringWriter);
+    json.beginObject();
+    systemInfoWriter.write(json);
+    json.endObject();
+    IOUtils.write(stringWriter.toString(), response.stream().output());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this use of "write" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1943)